torch/nn/functional.py

```diff
@@ -639,6 +639,15 @@ def pad(input, pad, mode='constant', value=0):
     else:
         raise NotImplementedError("Only 4D and 5D padding is supported for now")
 
+def center_crop(x, *args):
+    ndim = len(x.size()[2:])
+    if ndim == 1:
+        return Crop1d(args)(x)
```

Comment:
This is opposite to what is currently done in `nn`.
Usually we define the function in `functional`, and then call it in `modules`, but here it's done the other way around.
Maybe it might be good to unify it?